.github/workflows/test_on_push.yml

```
@@ -177,6 +177,49 @@ jobs:
       - name: Upload coverage report
         uses: codecov/codecov-action@v3.1.4
 
+  test_install_odes:
+    needs: style
+    runs-on: macos-latest
+    strategy:
```

```suggestion
    strategy:
        matrix:
            os: [ubuntu-latest, macos-latest]
            python-version: ["3.8", "3.9", "3.10", "3.11"]
```

We can keep it to just Python 3.11 for GNU/Linux and macOS later after we add this to the scheduled tests workflow at the time of completion